web/obs/src/stores/user.ts

```diff
@@ -15,6 +16,15 @@ export const useUser = defineStore('user', () => {
     if (data.value?.isLogin !== true) {
       data.value = undefined
     }
+
+    const resRoom = await client.GET('/bili/live/infoByUid', {
+      params: {
+        query: {
+          userId: userId.value!,
+        },
+      },
+    })
+    roomId.value = resRoom.data?.roomid ?? undefined
```

Comment:
你猜怎么着？我有非空断言洁癖 我看到 ! 就不自在

```suggestion
    if (!userId.value) return
    const resRoom = await client.GET('/bili/live/infoByUid', {
      params: {
        query: {
          userId: userId.value,
        },
      },
    })
    roomId.value = resRoom.data?.roomid ?? undefined
```